packages/core/EventManager.ts

```diff
@@ -371,7 +371,7 @@ export default class EventManager {
    * @private
    */
 
-  private getVideoCredential(event: CalendarEvent): CredentialWithAppName | undefined {
+  private async getVideoCredential(event: CalendarEvent): Promise<CredentialWithAppName | undefined> {
```

Comment:
Why is it declared async now without any change in functionality?

Reply:
I did have locations enabled for this PR, but we realized it will take more design work. There are some remnants of that functionality that I may have missed.